R/Spectra.R

```diff
@@ -1,4 +1,115 @@
-#' saveMsObject/readMsObject methods and related helper functions for
-#' Spectra objects
+#'@include PlainTextParam.R
+#'@title Methods to save and load contents of a Spectra object
 #'
+#' @description
+#'
+#' For `Spectra`, the exported data and related text files are:
+#'
+#' - The backend's [spectraData()] stored in a tabular format in a text file
```

Comment:
I would not describe this here - I would refer to the backend documentation for this file(s).